src/components/charts/other.js

```diff
@@ -179,6 +184,44 @@ export default class Other extends Component {
   }
 
   updatePolarConfiguration() {
+    const { polar } = this.state.configurations;
+    const { options } = this.state;
+    const circleLableFormatter = {
+      formatter: function () {
+          return this.value + '°';
+      }
+    };
+
+    options.title.text = polar.title ? 'Randomly generated data' : null;
+    options.subtitle.text = polar.title ? 'This data is not real' : null;
+    options.legend.enabled = polar.legend;
+    options.tooltip.enabled = polar.tooltip;
+    options.plotOptions.series.dataLabels.enabled = polar.dataLabels;
+    options.plotOptions.series.animation = polar.animation;
+    options.plotOptions.series.pointStart = polar.spiderMode ? undefined : 0;
+    options.plotOptions.series.pointInterval = polar.spiderMode
+      ? undefined
+      : 360 / options.series[0].data.length;
+    options.xAxis.tickInterval = polar.spiderMode
+      ? undefined
+      : 360 / options.series[0].data.length;
+    options.xAxis.categories = polar.spiderMode
+      ? options.series[0].data.map((item, i) => `Random Category ${i + 1}`)
+      : undefined;
+    options.xAxis.min = polar.spiderMode ? undefined : 0;
+    options.xAxis.max = polar.spiderMode ? undefined : 360;
+    options.xAxis.labels = polar.spiderMode ? {} : circleLableFormatter;
+    options.yAxis.gridLineInterpolation = polar.spiderMode ? 'polygone' : 'circle';
+    options.series.forEach((serie) => {
```

Comment:
I have to be careful here because I depend on the labels of the Select tag... 
Maybe is a good idea to change it in the future to some constants or at least to write a comment about it in the code...